dev/cognitiveservices/data-plane/Language/questionanswering-authoring.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "Microsoft Cognitive Language Service - Question Answering - Authoring",
     "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.",
-    "version": "2022-07-01-preview"
+    "version": "2022-10-01-preview"
```

Comment:
2022-10-01-preview is already being published in the specification/ folder. Is this actually for 10-01-preview, or does someone need to decide on the next api-version (it's largely arbitrary) for all services?

/cc @peytonfraser @robch

Reply:
This is actually 2022-10-01-preview for Language Authoring (the change here in QnA authoring is unintended as a result of a find and replace all). Shall I target my changes to the specification folder instead?

Reply:
Nvm saw the following comment, will create a new PR into kayousef/language/2022_10_01_preview